designs/developer/topology-45/index.md

```diff
@@ -0,0 +1,298 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Topology
+The Topology page is a primary navigation item in the OpenShift Developer perspective.
+
+## Topology
+The Topology view would be used in OpenShift Dev-Console to show semantic aggregated metadata about related pods, deployments, builds of application components in a namespace.  
+
+![Topology View](../topology-43/img/Topology_View.png)
+
+### No Project in Context
+When no project is selected, Topology View will prompt users to select from a given list of projects.  
+
+![No project in context](../topology-43/img/no_project_exists.png)
+
+#### No Workload Exists
+When no workload exists, the topology view redirects users to the add flows to start creating new workloads to be shown on the topology canvas.
+![Topology View](../topology-43/img/no_workload_exists.png)  
+
+### Application selector
+Users could choose to select either one of the applications or all the applications from the application selector.
+
+## Topology View
+
+
+### 1. Navigation item:
+'Topology' under the Developer perspective of OpenShift.
+
+### 2. Masthead
+The masthead in topology view includes the project and application selector.
+
+### 3. Topology Canvas
+This is the area where the workloads are visually placed in the Topology view.
+
+### 4. Toolbar
+The toolbar consists of the following controls:
+Zoom-in
+Zoom-out
+Fit to Screen
+Reset View
+
+### 5. Side Panel
+On selecting any workload, connectors or application group on Topology View, their respective details appear on a side panel on the right hand side.
+
+### 6. Nodes
+![Node design](../topology-43/img/node_design.png)
+
+Resources of type Deployments, Deployment Configs & Image Streams & Stateful Sets are shown as circle nodes in the topology view.
+
+The circle node can include a pod status indicator, an image,  decorators and a node label.  
+
+#### Pod Donut
+
+![Pod Status](../topology-43/img/pod_status.png)
+
+The pod donut represents the current status of the pods associated with the workload. If there are pods with different statuses, the pod donut is broken up into segments representing the status of the pods.
+
+The pod donut is segmented by status. Between each segment there’s a space of 3px.
+
+The name label contains a badge before the name, that denotes the type of resource.  
+
+#### Node Image
+The node image consists of the build image for the resource.  
+
+#### Decorators
+
+##### Route Decorator
+Clicking on the decorator opens the application in a new browser tab.
+
+##### Code Decorator
+The code decorator is located on the bottom right quadrant of the node.
+Go To GitHub or CHE Workspace  ** NEW IN 4.3
```

Comment:
looks like this was copied from 4.3 ... do we still want to call this out here ?

Reply:
Fixed